pyiron_base/storage/datacontainer.py

```diff
@@ -47,16 +47,16 @@ def _normalize(key):
     return key
 
 
-class DataContainer(MutableMapping, Lockable, HasGroups, HasHDF):
+class DataContainerBase(MutableMapping, Lockable, HasGroups):
```

Comment:
If we move this part off to its own repo (e.g. `pyiron/snippets`) and leave the child class here, then in the other location I would simply call this `DataContainer` and here do like `from pyiron.snippets import DataContainer as DataContainerBase`. In the context of making the break-apart go as smoothly as possible, I am totally behind the naming scheme used in this PR, I just want to double check that we have the same middle-term vision on naming here.

Reply:
Yeah, that's exactly how I thought it would go down.